client/state/inline-help/reducer.js

```diff
@@ -46,6 +51,16 @@ export const search = createReducer(
 				},
 			} );
 		},
+		[ INLINE_HELP_SELECT_RESULT ]: ( state, action ) => {
+			if ( ! isEmpty( state.items[ state.searchQuery ] ) ) {
```

Comment:
I think we could be more succinct here with something like:

```js
... ( state, action ) => ( { 
	...state,
	selectedResult: isEmpty( state.items[ state.searchQuery ] ) ? -1 : action.resultIndex,
} )
```

What do you think?

Not a blocker